### PR TITLE
Expanding SSI documentation (bug # 63184)

### PR DESCRIPTION
**Pull request by nightwatchcyber**
Fixing:
https://bz.apache.org/bugzilla/show_bug.cgi?id=63184

### CODE REVIEW

**Comment by markt-asf** [comment]
Close unintentionally during svn->git migration.

File: webapps/docs/ssi-howto.xml [hunk 1]
```diff
@@ -146,27 +146,53 @@ false.</li>
...
 </li>
 <li>
 <strong>flastmod</strong> - <code>&lt;!--#flastmod file=&quot;filename.shtml&quot; --&gt;</code>
-Returns the time that a file was lost modified.
+Returns the time that a file was lost modified, this is interpreted relative to the current directory.
+</li>
+<li>
+<strong>flastmod</strong> - <code>&lt;!--#flastmod virtual=&quot;filename.shtml&quot; --&gt;</code>
+Returns the time that a file was lost modified, this is interpreted relative to the document being served.
 </li>
```
Comment (markt-asf) [inline webapps/docs/ssi-howto.xml]: "last modified" not "lost modified"

File: webapps/docs/ssi-howto.xml [hunk 2]
```diff
@@ -146,27 +146,53 @@ false.</li>
...
 <li>
 <strong>include</strong> -  <code>&lt;!--#include virtual=&quot;file-name&quot; --&gt;</code>
-inserts the contents
+inserts the contents, this is interpreted relative to the document being served.
 </li>
 <li>
 <strong>flastmod</strong> - <code>&lt;!--#flastmod file=&quot;filename.shtml&quot; --&gt;</code>
-Returns the time that a file was lost modified.
+Returns the time that a file was lost modified, this is interpreted relative to the current directory.
+</li>
```
Comment (markt-asf) [inline webapps/docs/ssi-howto.xml]: "last" not "lost"
"relative to the current directory"?

File: webapps/docs/ssi-howto.xml [hunk 3]
```diff
@@ -146,27 +146,53 @@ false.</li>
...
+<li>
+<strong>flastmod</strong> - <code>&lt;!--#flastmod virtual=&quot;filename.shtml&quot; --&gt;</code>
+Returns the time that a file was lost modified, this is interpreted relative to the document being served.
 </li>
 <li>
 <strong>fsize</strong> - <code>&lt;!--#fsize file=&quot;filename.shtml&quot; --&gt;</code>
-Returns the size of a file.
+Returns the size of a file, this is interpreted relative to the current directory.
+</li>
```
Comment (markt-asf) [inline webapps/docs/ssi-howto.xml]: "relative to the current directory"? That looks like a copy/paste error.

File: webapps/docs/ssi-howto.xml [hunk 4]
```diff
@@ -146,27 +146,53 @@ false.</li>
...
+<li>
+<strong>fsize</strong> - <code>&lt;!--#fsize virtual=&quot;filename.shtml&quot; --&gt;</code>
+Returns the size of a file, this is interpreted relative to the document being served.
 </li>
```
Comment (markt-asf) [inline webapps/docs/ssi-howto.xml]: "relative to the document"?

File: webapps/docs/ssi-howto.xml (hunk 1 above)
Comment (nightwatchcyber) [reply webapps/docs/ssi-howto.xml]: fixed

File: webapps/docs/ssi-howto.xml (hunk 2 above)
Comment (nightwatchcyber) [reply webapps/docs/ssi-howto.xml]: Meaning its not a virtual path, I change the language to clarify

File: webapps/docs/ssi-howto.xml (hunk 3 above)
Comment (nightwatchcyber) [reply webapps/docs/ssi-howto.xml]: Meaning its not a virtual path, I change the language to clarify

File: webapps/docs/ssi-howto.xml (hunk 4 above)
Comment (nightwatchcyber) [reply webapps/docs/ssi-howto.xml]: Meaning its not a virtual path, I change the language to clarify

**Comment by nightwatchcyber** [comment]
I updated the pull request - let me if that works

**Comment by markt-asf** [comment]
Looks like you merged in changes from trunk rather than rebasing your PR and doing a forced push. I'll see if I can unpick this.

**Comment by markt-asf** [comment]
Applied manually.

**Pull request closed without merging**